modules/arm_plugin/src/arm_converter/arm_converter_ctc_greedy_decoder_seq_len.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2020-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+
+#include "arm_converter/arm_converter.hpp"
+#include <ngraph/runtime/reference/ctc_greedy_decoder_seq_len.hpp>
+
+namespace ArmPlugin {
+template<> Converter::Conversion::Ptr Converter::Convert(const opset::CTCGreedyDecoderSeqLen& node) {
+    auto make = [&] (auto refFunction) {
+        return this->MakeConversion(refFunction,
+                                    node.input(0),
+                                    node.input(1),
+                                    node.input(2),
+                                    node.output(0),
+                                    node.output(1),
+                                    node.get_input_shape(0),
+                                    node.get_output_shape(0),
+                                    node.get_merge_repeated());
+    };
+
+    switch (node.get_input_element_type(0)) {
```

Comment:
It would be nice to replace `switch` with `CallSwitch` as in 
https://github.com/openvinotoolkit/openvino_contrib/blob/master/modules/arm_plugin/src/arm_converter/arm_converter_one_hot.cpp#L56